### ISSUE TITLE
Switch to Xtext 2.12.0

### ISSUE THREAD

**Issue by kittaakos** [TypeFox/xtext-jflex#1]
Currently building with Maven fails.

We either need to switch to Xtext 2.12.0-SNAPSHOT or switch from `snapshots` to `releases` in the repository section in the POM.

I can confirm, switching to Xtext 2.12.0-SNAPSHOT works fine.

```
mvn clean verify
[INFO] Scanning for projects...
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for io.typefox:jflex-fragment:jar:0.0.1-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-compiler-plugin is missing. @ line 21, column 12
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building jflex-fragment 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for org.eclipse.xtend:xtend-maven-plugin:jar:2.11.0-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.148 s
[INFO] Finished at: 2017-02-16T16:36:18+01:00
[INFO] Final Memory: 8M/245M
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.eclipse.xtend:xtend-maven-plugin:2.11.0-SNAPSHOT or one of its dependencies could not be resolved: Failure to find org.eclipse.xtend:xtend-maven-plugin:jar:2.11.0-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of oss-snapshots has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

**Comment by kittaakos** [comment]
PR: https://github.com/TypeFox/xtext-jflex/pull/2

**Comment by kittaakos** [comment]
Resolved via #2.
